src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -97,7 +107,7 @@ export class HuiCardPicker extends LitElement {
       config = { ...config, ...cardConfig };
     }
 
-    this.cardPicked!(config);
```

Comment:
I read somewhere that callbacks are discouraged. Correct?

Reply:
Correct.